staging/src/k8s.io/apiserver/pkg/util/webhook/webhook.go

```diff
@@ -53,6 +61,14 @@ func NewGenericWebhook(registry *registered.APIRegistrationManager, codecFactory
 	if err != nil {
 		return nil, err
 	}
+
+	// Kubeconfigs can't set a timeout, this can only be set through a command line flag.
+	//
+	// https://github.com/kubernetes/client-go/blob/master/tools/clientcmd/overrides.go
+	//
+	// Set this to something reasonable so request to webhooks don't hang forever.
+	clientConfig.Timeout = requestTimeout
```

Comment:
@roycaihw could you confirm if this is the right way to set client timeout after your recent refactoring?

Reply:
nvm, I think it's still the correct API.